lib/output.js

```diff
@@ -511,6 +516,80 @@ function gif (options) {
   return this._updateFormatOut('gif', options);
 }
 
+/**
+ * Use these JP2 options for output image.
+ *
+ * @example
+ * // Convert any input to lossless JP2 output
+ * const data = await sharp(input)
+ *   .jp2({ lossless: true })
+ *   .toBuffer();
+ *
+ * @example
+ * // Convert any input to very high quality JP2 output
+ * const data = await sharp(input)
+ *   .jp2({
+ *     quality: 100,
+ *     chromaSubsampling: '4:4:4'
+ *   })
+ *   .toBuffer();
+ *
+ * @param {Object} [options] - output options
+ * @param {number} [options.quality=80] - quality, integer 1-100
+ * @param {boolean} [options.lossless=false] - use lossless compression mode
+ * @param {number} [options.tileWidth=256] - horizontal tile size
+ * @param {number} [options.tileHeight=256] - vertical tile size
+ * @param {string} [options.chromaSubsampling='4:4:4'] - set to '4:2:0' to use chroma subsampling
+ * @returns {Sharp}
+ * @throws {Error} Invalid options
+ */
```

Comment:
Might need to add `/* istanbul ignore next */` in here to avoid test coverage failure - see `gif()` for similar.